test/python/WMCore_t/Services_t/WorkQueue_t/WorkQueue_t.py

```diff
@@ -82,13 +82,13 @@ def testWorkQueueService(self):
         # This only checks minimum client call not exactly correctness of return
         # values.
         self.assertEqual(wqApi.getTopLevelJobsByRequest(),
-                         [{'total_jobs': 339, 'request_name': specName}])
+                         [{'total_jobs': 237, 'request_name': specName}])
```

Comment:
@amaltaro , is this 237 value appearing here and afterwards coming out from a calculation that involves the new job length? If yes, could we explicitly make the calculation here, instead of hardcoding the `total_job` value?

Reply:
Yes, it is possible. It depends on the input data and its metadata (number of events, number of lumis, time per event) and the target payload length.
I would have to figure out all the data in these test workflows, so I'd rather not make any of those changes as I don't think it is worth the extra effort - and now that all places have been identified, if we decide to change the target time again, it should be much easier to propagate the new values.

Reply:
Got it, I agree with your conclusion